src/Controller/CRUDController.php

```diff
@@ -1105,6 +1105,7 @@ protected function getBaseTemplate(): string
 
     /**
      * @throws \Exception
+     * @return string|null|void A custom error message to display in the flag bag instead of the generic one
```

Comment:
Please change to `@return string|null` and return null at the end of the method.